.github/workflows/enforce-license-compliance.yml

```diff
@@ -2,9 +2,18 @@ name: "CI: Enforce License Compliance"
 
 on:
   push:
-    branches: [master, develop, release/*]
+    branches:
+      - develop
+      - master
+      - v9
+      - v8
+      - release/**
```

Comment:
previously we didn't have a double ** here, mistake?

Reply:
hmm I just aligned this with the build flow, IMHO this is fine (worst case this covers more branches for this check which would be fine I guess)?